### ISSUE TITLE
[a11y] /advertising page is missing a main landmark

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15337]
# Axe-core® Accessibility Results for /en-US/advertising/

Page URL: https://www-dev.allizom.org/en-US/advertising/

## axe-core found 5 violations

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | Document should have one main landmark | landmark-one-main | Best practice | moderate | 1   |
| [2](#2) | All page content should be contained by landmarks | region | Best practice | moderate | 4   |

### Failed

1. Document should have one main landmark

[Learn more](https://dequeuniversity.com/rules/axe/4.10/landmark-one-main?application=playwright)

landmark-one-main

Best practice

Ensure the document has a main landmark

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>html<br>```<br><br>**Element source**<br><br>```<br><html class="windows js windows-10-plus x64 is-modern-browser mzp-has-sticky-navigation loaded" lang="en" dir="ltr" data-country-code="US" data-needs-consent="False" data-latest-firefox="131.0.3" data-esr-versions="128.3.1" data-gtm-container-id="GTM-MW3R8V" data-stub-attribution-rate="1.0" data-sentry-dsn="https://c1f94ae1b8c54ed0adcf223328157caf@o1069899.ingest.sentry.io/6260211"><br>``` | Fix all of the following:<br><br>*   Document does not have a main landmark |

2. All page content should be contained by landmarks

[Learn more](https://dequeuniversity.com/rules/axe/4.10/region?application=playwright)

region

Best practice

Ensure all page content is contained by landmarks

moderate

Issue Tags: cat.keyboard best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.mzp-c-callout<br>```<br><br>**Element source**<br><br>```<br><section class="mzp-c-callout mozads-header"><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |
| 2   | **Element location**<br><br>```<br>.mozads-t-dark<br>```<br><br>**Element source**<br><br>```<br><section class="mzp-c-split mozads-t-dark mzp-t-dark mzp-t-content-xl"><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |
| 3   | **Element location**<br><br>```<br>.mzp-l-split-reversed<br>```<br><br>**Element source**<br><br>```<br><section class="mzp-c-split mzp-l-split-reversed mzp-t-content-xl mozads-anonym"><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |
| 4   | **Element location**<br><br>```<br>#outer-wrapper > section:nth-child(4)<br>```<br><br>**Element source**<br><br>```<br><section><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |

**Comment by alexgibson** [comment]
Tip: all of the above violations can likely be fixed simply by wrapping the main page content in a `<main>` element.

**Comment by clarapenn** [comment]
@alexgibson could I be assigned this one, please? Thanks!

**Comment by stevejalim** [comment]
Fixed in 2dba421
